src/components/molecules/PlanRoom/Board/index.tsx

```diff
@@ -0,0 +1,18 @@
+import { Box, Image } from '@chakra-ui/react';
+import { DndProvider } from 'react-dnd';
+import { HTML5Backend } from 'react-dnd-html5-backend';
+import Items from 'components/molecules/PlanRoom/RoomTables/Table';
+import logo from 'components/molecules/PlanRoom/IMG/backGroundBoard.png';
```

Comment:
That should not be called `logo`.

Reply:
This is the path I will use for the photos -> src/assets/images/